### ISSUE TITLE
Global plasma power balance equation: energy conservation: revised

### ISSUE THREAD

**Issue by jonmaddock** [ukaea/PROCESS#131]
In GitLab by @mkovari on Sep 4, 2014, 15:30

Power balance code:    
```
if (iradloss == 1) then ! include pcoreradpv  
cc(i) = 1.0D0 - (ptrepv + ptripv + pcoreradpv) / (falpha*palppv + pchargepv + pohmpv + pinjmw/vol)  

else ! exclude pcoreradpv  
cc(i) = 1.0D0 - (ptrepv + ptripv) / (falpha*palppv + pchargepv + pohmpv + pinjmw/vol)  
```  

`ptrepv` and `ptripv` are calculated using the energy confinement time derived from the scaling law.  Thus although they are described as electron and ion transport powers, they actually include any power loss that is predicted by the scaling.  
`pcoreradpv` is the core radiation per unit volume including synchrotron.

My understanding is that the confinement scalings are all based on low radiation pulses.  Consequently it doesn't matter much whether or not the radiation was subtracted from the total energy loss from the plasma in the data - similar scalings would result.    A power plant will certainly be a high radiation machine - perhaps in the core (due to brem and synchrotron) and certainly in the edge due to impurity seeding.  The scaling data certainly do not predict this radiation - that needs to be done by the radiation model (which would otherwise be redundant).    However, if the transport is very "stiff", then the additional radiation *causes* an almost equal drop in transported power, leaving the confinement unchanged.  Since we don't know the truth, I suggest we leave this to the user, explaining very carefully what the options are.

Recommendations.

1.  Define a new quantity `pscaling = ptrepv + ptripv`

2.  Add a third option, and renumber the options as follows:
```
if (iradloss == 0) then                ! Total power lost is scaling power plus radiation
    cc(i) = 1.0D0 - (pscaling + pradpv) / (falpha*palppv + pchargepv + pohmpv + pinjmw/vol)  
else if (iradloss == 1) then        ! Total power lost is scaling power plus core radiation only  
    cc(i) = 1.0D0 - (pscaling + pcoreradpv) / (falpha*palppv + pchargepv + pohmpv + pinjmw/vol)  
else if (iradloss == 2) then       ! Total power lost is scaling power only, with no additional allowance for radiation.  
cc(i) = 1.0D0 - pscaling / (falpha*palppv + pchargepv + pohmpv + pinjmw/vol)  
```

3.  Explain this as follows in the User Guide.

**Effect of radiation on energy confinement**
Published confinement scalings are all based on low radiation pulses.  A power plant will certainly be a high radiation machine - both in the core, due to bremsstrahlung and synchrotron radiation, and in the edge due to impurity seeding.  The scaling data do not predict this radiation - that needs to be done by the radiation model.    However, if the transport is very "stiff", as predicted by some models, then the additional radiation *causes* an almost equal drop in power transported by ions and electrons, leaving the confinement nearly unchanged.  

To allow for these uncertainties, three options are available, using the switch `iradloss`.  In each case, the quantity `pscaling` is derived directly from the energy confinement scaling.

**iradloss = 0**  
Total power lost is scaling power plus radiation:  
`pscaling + pradpv = falpha*palppv + pchargepv + pohmpv + pinjmw/vol`

**iradloss = 1**  
Total power lost is scaling power plus core radiation only:  
`pscaling + pcoreradpv = falpha*palppv + pchargepv + pohmpv + pinjmw/vol`

**iradloss = 2**  
Total power lost is scaling power only, with no additional allowance for radiation.  This is not recommended for power plant models.
`pscaling = falpha*palppv + pchargepv + pohmpv + pinjmw/vol`

4.  Explain this as follows in the vardes file.

iradloss /1/ : switch for radiation loss term usage in power balance (see User Guide):  
   *  = 0       Total power lost is scaling power plus radiation  
   *  = 1       Total power lost is scaling power plus core radiation only    
   *  = 2       Total power lost is scaling power only, with no additional allowance for radiation.  This is not recommended for power plant models.

Comments from @hlux @pknight
